test/e2e/framework/util.go

```diff
@@ -5118,3 +5136,19 @@ func waitForServerPreferredNamespacedResources(d discovery.DiscoveryInterface, t
 	}
 	return resources, nil
 }
+
+// WaitForPersistentVolumeClaimBeRemoved waits for a PersistentVolumeClaim to be removed from the system until timeout occurs, whichever comes first.
+func WaitForPersistentVolumeClaimBeRemoved(c clientset.Interface, ns string, pvcName string, Poll, timeout time.Duration) error {
```

Comment:
this should be `WaitForPersistentVolumeClaimDeleted`, as `WaitForPersistentVolumeDeleted` or `WaitForNamespacesDeleted`.